.travis.yml

```diff
@@ -14,4 +14,4 @@ after_success:
   - npm run test:coverage && npm run test:coverage:upload
 branches:
   except:
-    - "/^v\\d+\\.\\d+\\.\\d+$/"
+    - "/^v\\d+\\.\\d+\\.\\d+$/"
```

Comment:
just a minor, we usually and each file with an empty line. Most of use use [`~/.editorconfig`](http://editorconfig.org/) for that, the configuration is called

```
insert_final_newline = true
```

That way it’s done automatically for you by your editor.